git diff libfive/bind/python/CMakeLists.txt
diff --git a/libfive/bind/python/CMakeLists.txt b/libfive/bind/python/CMakeLists.txt
index e60972e0..f46858d8 100644
--- a/libfive/bind/python/CMakeLists.txt
+++ b/libfive/bind/python/CMakeLists.txt
@@ -36,7 +36,7 @@ if(UNIX)
     endif()

     # Install all Python files into that directory
-    install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/libfive
-            DESTINATION ${PYTHON_SITE_PACKAGES_DIR}
-            FILES_MATCHING PATTERN "*.py")
+    #    install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/libfive
+    #        DESTINATION ${PYTHON_SITE_PACKAGES_DIR}
+    #        FILES_MATCHING PATTERN "*.py")
 endif(UNIX)
